spec/features/lettings_log_spec.rb

```diff
@@ -88,14 +88,115 @@
         click_button("Save and continue")
         log_id = page.current_path.scan(/\d/).join
         visit("lettings-logs/#{log_id}/setup/check-answers")
-        expect(page).to have_content("Stock owner #{support_user.organisation.name}")
+        expect(page).to have_content("Stock owner User org")
         expect(page).to have_content("You have answered 2 of 8 questions")
       end
     end
+
+    context "when the owning organisation question isn't answered" do
+      it "doesn't show the managing agent question" do
+        visit("/lettings-logs")
+        click_button("Create a new lettings log")
+        click_link("Set up this lettings log")
+        log_id = page.current_path.scan(/\d/).join
+        click_link("Skip for now")
+        expect(page).not_to have_current_path("/lettings-logs/#{log_id}/managing-organisation")
+      end
+    end
+
+    context "when the owning organisation question is answered" do
+      context "and the owning organisation doesn't hold stock" do
+        let!(:managing_org) { create(:organisation, name: "Managing org") }
+        let!(:org_rel) { create(:organisation_relationship, parent_organisation: support_user.organisation, child_organisation: managing_org) }
+
+        before do
+          support_user.organisation.update!(holds_own_stock: false)
+        end
+
+        it "shows the managing organisation question" do
+          visit("/lettings-logs")
+          click_button("Create a new lettings log")
+          click_link("Set up this lettings log")
+          log_id = page.current_path.scan(/\d/).join
+          select(support_user.organisation.name, from: "lettings-log-owning-organisation-id-field")
+          click_button("Save and continue")
+          expect(page).to have_current_path("/lettings-logs/#{log_id}/managing-organisation")
+          select(managing_org.name, from: "lettings-log-managing-organisation-id-field")
+          click_button("Save and continue")
+          visit("lettings-logs/#{log_id}/setup/check-answers")
+          expect(page).to have_content("Managing agent Managing org")
+          expect(support_user.organisation.managing_agents).to eq(org_rel.child_organisation)
```

Comment:
This line is pretty artificial/contrived way of explicitly referencing the organisation relationship in the test. Without it, Rubocop complains that something being defined with `let!` is not being explicitly used in the test. The reason I have defined the organisation relationship at all is because its existence influences the logic around the answer options for the question, and in particular whether the question gets auto-assigned an answer.